pkgs/servers/http/apache-modules/mod_tile/default.nix

```diff
@@ -1,77 +1,66 @@
-{ lib
+{ fetchFromGitHub
+, lib
 , stdenv
-, fetchFromGitHub
-, fetchpatch
-, autoreconfHook
 , apacheHttpd
 , apr
+, aprutil
+, boost
 , cairo
+, cmake
+, curl
+, glib
+, gnumake
+, gtk2
+, icu
 , iniparser
+, libmemcached
 , mapnik
-, boost
-, icu
-, harfbuzz
-, libjpeg
-, libtiff
-, libwebp
-, proj
-, sqlite
 }:
 
 stdenv.mkDerivation rec {
   pname = "mod_tile";
-  version = "unstable-2017-01-08";
+  version = "0.6.1-master";
 
   src = fetchFromGitHub {
     owner = "openstreetmap";
     repo = "mod_tile";
-    rev = "e25bfdba1c1f2103c69529f1a30b22a14ce311f1";
-    sha256 = "12c96avka1dfb9wxqmjd57j30w9h8yx4y4w34kyq6xnf6lwnkcxp";
+    rev = "f521540df1003bb000d7367a59ad612161eab0f0";
+    sha256 = "sha256-jIqeplAQt4W97PNKm6ZDGPDUc/PEiLM5yEdPeI+H03A=";
   };
 
-  patches = [
-    # Pull upstream fix for -fno-common toolchains:
-    #  https://github.com/openstreetmap/mod_tile/pull/202
-    (fetchpatch {
-      name = "fno-common";
-      url = "https://github.com/openstreetmap/mod_tile/commit/a22065b8ae3c018820a5ca9bf8e2b2bb0a0bfeb4.patch";
-      sha256 = "1ywfa14xn9aa96vx1adn1ndi29qpflca06x986bx9c5pqk761yz3";
-    })
+  nativeBuildInputs = [
+    cmake
+    gnumake
   ];
 
-  # test is broken and I couldn't figure out a better way to disable it.
-  postPatch = ''
-    echo "int main(){return 0;}" > src/gen_tile_test.cpp
-  '';
-
-  nativeBuildInputs = [ autoreconfHook ];
   buildInputs = [
     apacheHttpd
     apr
+    aprutil
+    boost
     cairo
+    curl
+    glib
+    gtk2
```

Comment:
`gtk2` is a very odd dependency to see in `apache` module. `gtk-2` being old and deprecated has numerous problems as a library. Is it needed here? Or maybe it could be disabled at build time?